clientd3d/client.rc

```diff
@@ -1785,6 +1785,15 @@ BEGIN
     IDS_SIGNUPMSG           "Your new account [%s] was successfully created!\n\nBefore you can login and play...\n\nCheck your e-mail (%s) to verify your account now."
 END
 
+STRINGTABLE
+BEGIN
+    IDS_RARITY_GRADE_MAGIC       "(magic)"

```

Comment:
The parentheses shouldn't be in here.  That makes it impossible to use the names without parentheses elsewhere.

Reply:
Moved the parentheses out of the `client.rc` file as requested.  They are now in the calling `snprintf` functions.